apps/entry/migrations/0027_auto_20200911_0725.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.1.15 on 2020-09-11 07:25
+
+from django.db import migrations
+
+
+def update_attributes(apps, schema_editor):
+    Attribute = apps.get_model('entry', 'Attribute')
+    for each in Attribute.objects.all():
+        each.save()
```

Comment:
This is a heavy migration. Let's not do this here (or only do for development).
For production, let's create a script in bulk_migration and run the command maunally.

Reply:
Also,

- change the Attribute filter so that only scale widget attributes are recalculated.
- use entry.utils.update_attributes module for updating.

Something like this (I am little rusty so make sure to validate)
```python
from entry.utils import update_atrributes
update_attributes(widget__widget_id__in=['scaleWidget'])
 ```